Add missing titles

atduarte opened the pull request:
On pages: extension-activation, extension-uninstall, enterprise-license

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/784"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/784"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>



<a href="https://at-prebuild-page.staging.gitpod-dev.com/#https://github.com/gitpod-io/website/pull/784"><img src="https://at-prebuild-page.staging.gitpod-dev.com/button/open-in-gitpod.svg"/></a>

mikenikles reviewed (changes requested):
Nice catch and thank you for the PR.

To ensure standardization across the website, please use the `<OpenGraph />` component. It takes care of the `<title>` and all necessary meta tags needed for social media previews.

See [this example](https://github.com/gitpod-io/website/blob/main/src/routes/features.svelte#L105).

atduarte commented:
> Nice catch and thank you for the PR.
> 
> To ensure standardization across the website, please use the `<OpenGraph />` component. It takes care of the `<title>` and all necessary meta tags needed for social media previews.
> 
> See [this example](https://github.com/gitpod-io/website/blob/main/src/routes/features.svelte#L105).

I saw open-graph but since these pages are not supposed to be shared or show up in search engines I thought it would be better to just add the title. Changed to the open-graph and added a `noindex` meta tag.
